packages/search/src/features/registration/assignment/service.ts

```diff
@@ -45,15 +50,16 @@ export async function updateEventToAddAssignment(requestBundle: Hapi.Request) {
     task,
     'http://opencrvs.org/specs/extension/regLastOffice'
   )
+  const regLastOffice =
+    regLastOfficeIdentifier &&
+    (await fetchLocation(
+      resourceIdentifierToUUID(regLastOfficeIdentifier.valueReference.reference)
+    ))
```

Comment:
This handler doesn't get the full bundle but only a subset of it, hence the need to fetch the location. I believe this could be refactored if we fetch the record in workflow from `/records/{id}` and send it here fully.